### ISSUE TITLE
CLI ERROR: 'unicodeescape' codec can't decode bytes in position 238-239: truncated \UXXXXXXXX escape

### ISSUE THREAD

**Issue by jeking3** [StackStorm/st2#4385]
##### SUMMARY

Running a command from the vSphere pack that executes a command on a windows guest and returns a string is successful, and I can see it in the st2 GUI, but the command line response indicates a problem handling the data.

##### ISSUE TYPE
 - Bug Report

##### STACKSTORM VERSION
st2 3.0dev (5377ebb), on Python 2.7.6

##### OS / ENVIRONMENT / INSTALL METHOD
st2-docker with ST2_IMAGE_TAG=dev

##### STEPS TO REPRODUCE
Requires a working vSphere environment.
Install the vSphere pack with guest extensions from my branch:
```
# st2 pack install https://github.com/jeking3/stackstorm-vsphere.git=guestops
```
Run a command against a guest, for example:
```
root@b9bab223e4fa:/# st2 run vsphere.guest_dir_create vm_id=vm-45 username=Administrator password=******** prefix=issue_ suffix=_test
.
ERROR: 'unicodeescape' codec can't decode bytes in position 238-239: truncated \UXXXXXXXX escape
```

##### EXPECTED RESULTS
The action output in the GUI is reasonable:
```
{
  "result": "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\jking_vmware20_test",
  "exit_code": 0,
  "stderr": "/opt/stackstorm/st2/lib/python2.7/site-packages/cryptography/hazmat/primitives/constant_time.py:26: CryptographyDeprecationWarning: Support for your Python version is deprecated. The next version of cryptography will remove support. Please upgrade to a 2.7.x release that supports hmac.compare_digest as soon as possible.\n  utils.DeprecatedIn23,\n",
  "stdout": ""
}
```
However the CLI choked.

##### ACTUAL RESULTS
```
ERROR: 'unicodeescape' codec can't decode bytes in position 238-239: truncated \UXXXXXXXX escape
```

**Comment by LindsayHill** [comment]
What does `st2 --debug execution get <ID>` return?

**Comment by LindsayHill** [comment]
_possibly_ related:
https://github.com/StackStorm/st2-docker/issues/129
https://github.com/StackStorm/st2/issues/4120
https://github.com/StackStorm/st2/pull/4127

**Comment by jeking3** [comment]
The tail end with the stack trace is probably what you are looking for.  If you need more let me know.
```
Traceback (most recent call last):
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2client/shell.py", line 402, in run
    func(args)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2client/commands/resource.py", line 47, in decorate
    return func(*args, **kwargs)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2client/commands/action.py", line 1229, in run_and_print
    return self._print_execution_details(execution=execution, args=args, **kwargs)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2client/commands/action.py", line 366, in _print_execution_details
    self.print_output(instance, formatter, **options)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2client/commands/__init__.py", line 90, in print_output
    output = self.format_output(subject, formatter, *args, **kwargs)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2client/commands/__init__.py", line 86, in format_output
    return func(subject, *args, **kwargs)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2client/formatters/execution.py", line 122, in format
    return strutil.unescape(str(output)).decode('unicode_escape').encode('utf-8')
UnicodeDecodeError: 'unicodeescape' codec can't decode bytes in position 251-252: truncated \UXXXXXXXX escape
```
I think it has something to do with backslashes.  I like the debug command - thanks. :)

**Comment by LindsayHill** [comment]
In this case I'd like to see the whole debug output if possible. I'm specifically interested in seeing what the API returns, since here the exception is coming from the client, not the server

**Comment by Kami** [comment]
I can reproduce this issue. Will push a fix shortly.

**Comment by Kami** [comment]
Bug fix in #4407 - thanks again for reporting this.

(I could easily reproduce this issue by running `st2 run run core.local cmd='echo C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\jking_vmware20_test'`)

**Comment by Kami** [comment]
Thanks again for reporting this.

Fix in #4407 has been merged and will be included in the next release. It slightly longer to merge it due to some issues with our CI system.
